examples/cluster-with-serviceiprange/Pulumi.yaml

```
@@ -0,0 +1,3 @@
+name: example-cluster-with-serviceIpRange
```

Super nitpick (sorry!):

```suggestion
name: example-cluster-with-serviceiprange
```